theme_tutorial/static/src/scss/style.scss

```diff
@@ -1,16 +1,29 @@
-.services {
-    background-color: #eaeaea;
-    padding: 1em;
-    margin: 2em 0 3em;
-
-    li{
-        display: block;
-        position: relative;
-        background-color: #16a085;
-        color: #fff;
-        padding: 2em;
-        text-align: center;
-        margin-bottom: 1em;
-        font-size: 1.5em;
-    }
-}
\ No newline at end of file
+.services {
+    background-color: #eaeaea;
+    padding: 1em;
+    margin: 2em 0 3em;
+
+    li {
+        display: block;
+        position: relative;
+        background-color: #16a085;
+        color: #fff;
+        padding: 2em;
+        text-align: center;
+        margin-bottom: 1em;
+        font-size: 1.5em;
+    }
+}
+
+.snippet_testimonial {
+    border: 1px solid #eaeaea;
+
+    &.opt_shadow img {
```

Comment:
Before create a new style try to use bootstrap default styles

![image](https://github.com/AntonioMartinD/Building_a_website/assets/35231827/47523062-ae22-49d6-9c3e-68bf3b133b7c)